packages/prisma/migrations/20230309203435_make_booking_and_workflow_step_optional_for_workflow_reminder/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- DropForeignKey
+ALTER TABLE "WorkflowReminder" DROP CONSTRAINT "WorkflowReminder_bookingUid_fkey";
+
+-- DropForeignKey
+ALTER TABLE "WorkflowReminder" DROP CONSTRAINT "WorkflowReminder_workflowStepId_fkey";
+
+-- AlterTable
+ALTER TABLE "WorkflowReminder" ALTER COLUMN "bookingUid" DROP NOT NULL,
+ALTER COLUMN "workflowStepId" DROP NOT NULL;
```

Comment:
Checked there are no NULL variables on production for `bookingUid` and `workflowStepId`